app/models/katello/glue/pulp/repo.rb

```diff
@@ -429,8 +429,8 @@ def puppet_module_ids
         Katello.pulp_server.extensions.repository.puppet_module_ids(self.pulp_id)
       end
 
-      def docker_image_ids
-        Katello.pulp_server.extensions.repository.docker_image_ids(self.pulp_id)
+      def docker_img_ids
```

Comment:
Can you confirm or find that in the Rails 4 docs? This is less of a problem, but `rpm_ids` was used because that is specifically tied to RPMs as we were moving away from the generic 'package' in light of debian support down the road.